transforms/ember-object/__testfixtures__/options/decorators-disabled.input.js

```diff
@@ -4,74 +4,74 @@ import { inject as service } from "@ember/service";
 import { on } from "@ember/object/evented";
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo1 = EmberObject.extend({
```

Comment:
I wonder why this isn't transformed.

Reply:
Decorators are disable in the config for this test

Reply:
Oh, I see. You are suggesting we find a way to transform w/o decorators.

That seems fair but outside the scope of our project. I'm curious why someone would disable decorators during the codemod TBH given that Ember Octane relies pretty heavily on them.